contrib/charts/cert-manager/templates/hook_create_issuers.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.clusterIssuer.email }}
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: create-issuers
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
+  template:
+    metadata:
+      name: create-issuers
+      namespace: {{ .Release.Namespace }}
+    spec:
+      restartPolicy: Never
+      containers:
+      - name: create-issuers
+        image: claranet/gcloud-kubectl-docker
+        imagePullPolicy: Always
+        command: ["kubectl","create","-f","/tmp/"]
+        volumeMounts:
+          - name: issuers
+            mountPath: /tmp
+      volumes:
+        - name: issuers
+          configMap:
+            name: issuers
+      serviceAccountName: {{ template "cert-manager.serviceAccountName" . }}
```

Comment:
This service account needn't be the same as cert-manager itself - can we create a specialised RBAC policy here just for the installer?